### DISCUSSION TITLE
Unsure how to set `title` to an empty string without the default being applied

### DISCUSSION THREAD

**Discussion by cmpadden** [ghostty-org/ghostty#3039] [Q&A] [answered]
Upon installation, one of the first things I attempted to do was set an empty string for the title bar. Here are the steps I followed in attempting to do this.

1. I referenced the _Configuration Documentation_ section of the README
    * Choosing to use `ghostty +show-config --default --docs`
    * Realizing I didn't have the binary on my `PATH`, I modified the command to the following:
        - `/Applications/Ghostty.app/Contents/MacOS/ghostty +show-config --default --docs`
2. From the configuration reference, I searched for `title` and found what I was looking for:
    ```
    # The title Ghostty will use for the window. This will force the title of the
    # window to be this title at all times and Ghostty will ignore any set title
    # escape sequences programs (such as Neovim) may send.
    # 
    # This configuration can be reloaded at runtime. If it is set, the title
    # will update for all windows. If it is unset, the next title change escape
    # sequence will be honored but previous changes will not retroactively
    # be set. This latter case may require you restart programs such as neovim
    # to get the new title.
    title = 
   ``` 
3. I tried setting `""` and `''` with no luck
4. Searching Discord, I found one user suggesting `macos-titlebar-style = hidden`, and ended up going with this to hide the bar entirely

I'm still unsure on how set the `title` to an empty string. Hopefully I'm not missing something obvious, but wanted to share my "new user experience".

Thanks!

**Comment by mitchellh** [comment, marked as answer]
I believe setting it to spaces will work. `" "`. If that works we should document it.

**Reply by cmpadden** [reply]
`title = " "` did indeed work--should've tried that!

thanks, @mitchellh .

**Reply by cmpadden** [reply]
You can find a pull request here to mention it in the config docstring: https://github.com/ghostty-org/ghostty/pull/3045
